### PR TITLE
Ready for 0.2.0 release: remove `conda_build_config.yaml`

### PR DESCRIPTION
**Pull request by cadenmyers13**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by cadenmyers13** [comment]
@conda-forge-admin, please rerender

File: README.md [hunk 1]
```diff
@@ -39,14 +39,14 @@ Current release info
 Installing diffpy.fourigui
 ==========================
 
-Installing `diffpy.fourigui` from the `conda-forge/label/diffpy.fourigui_rc` channel can be achieved by adding `conda-forge/label/diffpy.fourigui_rc` to your channels with:
+Installing `diffpy.fourigui` from the `conda-forge` channel can be achieved by adding `conda-forge` to your channels with:
 
 ```
-conda config --add channels conda-forge/label/diffpy.fourigui_rc
```
Comment (bobleesj) [inline README.md:45]: nice, no more `rc`

File: recipe/conda_build_config.yaml [hunk 2]
```diff
@@ -1,2 +0,0 @@
-channel_targets:
-  - conda-forge diffpy.fourigui_rc
```
Comment (bobleesj) [inline recipe/conda_build_config.yaml:2]: good

**Comment by bobleesj** [comment]
@cadenmyers13 for bookkeeping purpose, maybe title like here you've done: https://github.com/conda-forge/diffpy.fourigui-feedstock/pull/3 -> "Ready for 0.2.0 release:  remove `conda_build_config.yam`"

**Comment by bobleesj** [comment]
This is great. 

I will merge it since the only change is making it non `rc` and no `meta.yaml` has been modified.

**Comment by bobleesj** [comment]
It will take like ~30 mins to be deployed on conda-forge... so if you could test it after lunch or sth

**Comment by cadenmyers13** [comment]
@bobleesj will do. Then will close 0.2.0 release issue

**Pull request merged**